docs/components/Colors.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --color-width: 177px;
+  --color-height: 60px;
+  --hex-font-size: 11px;
+  --border-color: #f0f0f0;
+  --color-text-white: #fefefe;
+  --color-text-gray: var(--accent-dark);
+}
+
+.Colors {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px var(--space-medium);
+}
+
+.Color {
```

Comment:
I'm thinking every color tile needs a border around it to better associate the color variable with the name and to help prevent "blending" with the background color when there's a non-contrasting background, something like what's displayed below: 

![gray-40 border around the color tile](https://github.com/dequelabs/cauldron/assets/1062039/c2ab8a70-a0ba-4729-a822-d1af826ed92f)